external/os-autoinst-common/tools/update-deps

```diff
@@ -69,8 +69,9 @@ sub update_dockerfile {
     my $end = '# AUTODEPS END';
     my $run = <<"EOM";
 # This part is autogenerated by $scriptname from $yamlfile
+# hadolint ignore=DL3034,DL3037
 RUN zypper in -y -C \\
-$dep   && true
+$dep   && zypper clean
```

Comment:
I thought we had this in already but maybe I got confused and in the end the according subrepo pull maybe has been only applied for os-autoinst so far.